src/GitLab/Types.hs

```diff
@@ -244,6 +248,30 @@ data RepositoryTree = RepositoryTree
   , repositoryTreeMode :: Text
   } deriving Show
 
+data CommitActionType = ActionCreate
+                      | ActionDelete
+                      | ActionMove
+                      | ActionUpdate
+                      deriving Show
```

Comment:
nitpick: Can we format code so there is no alignment that relies on length of a symbol? In this case

```haskell
data CommitActionType
  = ActionCreate
  | ActionDelete
  | ActionMove
  | ActionUpdate
  deriving Show
```

This format doesn't need to modify indentation even if we changed `CommitActionType` to `VeryLongNameThatWasPreviouslyCommitActionType`.